.github/scripts/run-migration.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "Build Lib9c.Tools"
```

Comment:
Informational log messages should go to the standard error (rather than the standard output):

```suggestion
echo "Build Lib9c.Tools" > /dev/stderr
```